config/configuration.md

```diff
@@ -201,6 +201,19 @@ The global coefficient applied to all deviation checks. Used to differentiate al
 
 Defaults to `1`.
 
+#### `asyncBeaconUpdatedeviationThresholdFactor` _(optional)_
+
+The global deviation factor applied to async beacon updates deviation checks. When configured, this value will be used
+on beacons sets that don't require an update but some of its beacon constituents do. So if a beacon set has a 1%
+deviation threshold, Airseeker will only update its beacons when deviation of each beacon exeeds (1 \*
+asyncBeaconUpdatedeviationThresholdFactor)%. For example:
+
+```jsonc
+"asyncBeaconUpdatedeviationThresholdFactor": 5,
+```
+
+Defaults to `1`.
```

Comment:
You're lying 😄 .

Reply:
https://github.com/api3dao/airseeker/pull/343/commits/15835695ec8e9e5e6d64f2e11d20c0910e9aec9d